src/util/analytics-client.ts

```diff
@@ -0,0 +1,91 @@
+import {getLogger} from "../config/logger";
+import { omit } from "lodash";
+import { optionalRequire } from "optional-require";
+import {AnalyticsConfigEnum} from "../interfaces/common";
+import {isNodeProd} from "./isNodeEnv";
+
+const { analyticsClient } = optionalRequire("@atlassiansox/analytics-node-client") || {};
+const logger = getLogger("analytics")
+const instance = process.env.INSTANCE_NAME;
+const appKey = `com.github.integration${instance ? `.${instance}` : ""}`;
+
+function sendAnalytics(eventType: "trait")
+function sendAnalytics(eventType: "screen", attributes:{name:string} & Record<string,unknown>)
+function sendAnalytics(eventType: "ui" | "track" | "operational", attributes: Record<string,unknown>)
+function sendAnalytics(eventType: string, attributes?: Record<string, unknown>): void {
+	if(!analyticsClient && !isNodeProd()){
+		return;
+	}
+
+	const analyticsNodeClient = analyticsClient({
+		env: AnalyticsConfigEnum.ProdEnv,
+		product: AnalyticsConfigEnum.Product
+	});
+
+	const baseAttributes = {
+		userId: "anonymousId",
+		userIdType: "atlassianAccount",
+		tenantIdType: "cloudId",
+		tenantId: "NONE",
+	}
+
+	attributes!.appKey = appKey;
+
+	switch (eventType) {
+		case "screen":
+			wrapPromises(analyticsNodeClient.sendScreenEvent({
+				...baseAttributes,
+				name: attributes?.name,
+				screenEvent: {
+					platform: "web",
+					attributes: {
+						...omit(attributes, "name"),
+						appKey
+					}
+				}
+			}))
+			break;
+		case "ui":
+			wrapPromises(analyticsNodeClient.sendUIEvent({
+				...baseAttributes,
+				uiEvent: {
+					attributes: attributes
```

Comment:
/nit don't need to do `attributes: atributes`, eslint will complain about this.  Just a single `attributes` will do the key and the value :)